experimenter/experimenter/experiments/api/v5/serializers.py

```diff
@@ -1652,39 +1631,51 @@ def _validate_localizations(self, data):
                 }
             )
 
-        localizations_json = data.get("localizations")
         experiment_locales = data.get("locales")
-
         if not experiment_locales:
             raise serializers.ValidationError(
                 {"locales": ["Locales must not be empty for a localized experiment."]}
             )
 
-        experiment_locale_codes = [locale.code for locale in experiment_locales]
-        localizations = json.loads(localizations_json)
+        localizations_json = data.get("localizations")
+        try:
+            localizations = json.loads(localizations_json)
+        except Exception as e:
+            raise serializers.ValidationError(f"Invalid JSON: {e}")
```

Comment:
Can you catch a ValidationError only? If not, I'd suggest using ```from e``` so we don't lose the context of the actual error.